docs/validator/00-rewards.md

```diff
@@ -0,0 +1,20 @@
+---
+id: rewards
+title: Validator rewards
+sidebar_label: Rewards
+---
+
+## Guaranteed rewards
+
+> IOV will guarantee up to *5000 IOV tokens per month for two years* for up to *15 validators* starting from mainnet launch.
+
+The realized guaranteed monthly reward to a validator is the product of the 5000 IOV tokens and the given validator's quality score relative to the highest quality score.  So, if your quality score is 87% and the highest quality score is 91% then your realized guaranteed reward is `87% / 91% * 5000 IOV tokens = 4780 IOV tokens`.
+
+> Guaranteed rewards are not inflationary.  They are a pre-allocated subsidy of IOV tokens created at genesis that is distributed in an algorithmic manner that decreases to zero over time.
+
+Realized guaranteed rewards are not the only reward for which validators can qualify because validators can be depositors, too.
+
+## Depositor rewards
+
+The IOV token economy has the notion of <a href="https://docs.google.com/spreadsheets/d/1zfxAlOXym3S5-yARBIVyJyxGkqq1a-8Tg2jh4L2lLD0" target="_blank">term deposits</a>, particularly to compensate early backers of our efforts while chain turnover grows.  The deposit rate for non-validators is set by the chain's governance board, but for validators the <a href="https://medium.com/iov-internet-of-values/iov-token-economic-cycle-distribution-of-the-bns-turnover-11c4cec202a8" target="blank_">deposit rate is their quality score</a>, which is much higher than the non-validator deposit rate for performant validators.  <a href="https://www.iov.one/#token-timeline-block-section" target="blank_">The IOV token sale is still open to validators</a>, so take advantage of it so that you can make a term deposit for additional rewards.
```

Comment:
For links, markdown has a sweet feature such as: \[link title\]\(link\). No need to write html links here.

Reply:
I need the target=blank_ feature of html, though.  I don't want to replace the current page with the page pointed to by the link. The search I did claimed that target=blank_ is not an option in markdown.  Was I misinformed?

Reply:
You're right. Could not find any solution that works with docusaurus. Approving :+1: